UI/Pages/WatchlistPage.cs

```diff
@@ -0,0 +1,23 @@
+﻿using OpenQA.Selenium;
+using System.Web.UI.WebControls;
+using UI.Utils;
+
+namespace UI.Pages
+{
+	public class WatchlistPage : AbstractPage
+	{
+		private static readonly By viewWatchlistTab = By.XPath("(//a[@href='/wiki/Special:EditWatchlist'])[1]");
```

Comment:
Please avoid appeal to element directly ([1]). Try to make better path without this. If you will not, then ask me